nilearn/plotting/surf_plotting.py

```diff
@@ -54,6 +58,9 @@ def plot_surf(surf_mesh, surf_map=None, bg_map=None,
         which is a name of a matplotlib colormap, or a matplotlib
         colormap object. If None, matplolib default will be chosen
 
+    colorbar : bool, optional, default is False
+        If True, a colorbar is added.
```

Comment:
- Could you add colorbar is displayed as symmetric by default. 
- It displays colorbar of surface map